e2e/scripts/st_heading.py

```diff
@@ -16,3 +16,13 @@
 
 st.header("This header is awesome!")
 st.header("This header is awesome too!", anchor="awesome-header")
+
+st.title("`Code` - Title without Anchor")
```

Comment:
nit: Just saw that all titles have anchor. It is just that you can configure a custom anchor with the `anchor` parameter. Maybe just remove the `without Anchor` here, so just: `Code` - Title . Same of the Subheader below.